Source/WebCore/Modules/applepay/paymentrequest/ApplePayPaymentHandler.cpp

```diff
@@ -888,7 +888,11 @@ ExceptionOr<void> ApplePayPaymentHandler::retry(PaymentValidationErrors&& valida
     if (errors.isEmpty())
         errors.append(ApplePayError::create(ApplePayErrorCode::Unknown, std::nullopt, nullString()));
 
+#if ENABLE(APPLE_PAY_PAYMENT_ORDER_DETAILS)
+    ApplePayPaymentAuthorizationResult authorizationResult { ApplePayPaymentAuthorizationResult::Failure, WTFMove(errors), std::nullopt };
+#else
```

Comment:
Rather than repeat this twice, how about we separate this onto separate lines as that would let us avoid having to even care about `ENABLE(APPLE_PAY_PAYMENT_ORDER_DETAILS)` entirely?
```
ApplePayPaymentAuthorizationResult authorizationResult;
authorizationResult.status = ApplePayPaymentAuthorizationResult::Failure;
authorizationResult.errors = WTFMove(errors);
```